src/test/java/com/savvato/tribeapp/services/UserRoleServiceImplTest.java

```diff
@@ -40,3 +40,4 @@ public void getRoles() {
     }
 
 }
+  
```

Comment:
Hmm... I was trying to remove this blank line.. so you may see some commits from me attempting that.. never mind, however.